packages/venia-concept/src/components/Icon/icon.js

```diff
@@ -1,10 +1,51 @@
 import React, { Component } from 'react';
-import Feather from 'feather-icons';
 import PropTypes from 'prop-types';
 
 import classify from 'src/classify';
 import defaultClasses from './icon.css';
 
+// TODO: Tree-shaking is broken in react-feather right now, see the issue below.
+//   When this is fixed you can import without using full path.
+//   https://github.com/carmelopullara/react-feather/issues/6
```

Comment:
If we want to avoid having to do full path imports we could switch to [react-icons](https://www.npmjs.com/package/react-icons). I guess the icon font lib we use is really a UX call? @soumya-ashok any opinions?